DB Setup/data_insertion.sql

```diff
@@ -0,0 +1,83 @@
+
+\c va_trial
+
+insert into SessionKeys values
+	(0, 00001, '6:00AM', '6:30AM'),
+	(1, 00002, '7:00AM', '7:30AM'), (2, 00003, '8:00AM', '8:30AM'),
+	(3, 00004, '9:00AM', '9:30AM'),
+	(4, 00005, '10:00AM', '10:30AM'),
+	(5, 00006, '11:00AM', '11:30AM'),
+	(6, 00007, '12:00PM', '12:30PM');
+
+insert into tfiSurvey values
+	(10000, TRUE,  00001, 'June 1, 2019', '6:30AM', 0, 2, 3, 5, 3, 5, 1, 4, 7),
+	(20000, TRUE, 00002, 'June 2, 2019', '7:30AM', 1, 9, 8, 9, 3, 4, 3, 4, 8),
+	(30000, TRUE, 00003, 'June 3, 2019', '8:30AM', 0, 1, 5, 3, 2, 5, 0, 1, 5),
+	(40000, FALSE, 00004, 'June 4, 2019', '9:30AM', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL),
+	(50000, TRUE, 00005, 'June 5, 2019', '10:30AM', 0, 8, 3, 4, 5, 6, 1, 2, 6),
+	(60000, TRUE, 00006, 'June 6, 2019', '11:30AM', 1, 2, 3, 4, 5, 6, 7, 8, 9),
+	(70000, FALSE, 00007, 'June 7, 2019', '12:30PM', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
+
+insert into thsSurvey values
+	(10000, 00001, 'June 1, 2019', '6:30AM', 0, 4, 3, 'example'),
+	(20000, 00002, 'June 2, 2019', '7:30AM', 1, 5, 8, 'example'),
+	(30000, 00003, 'June 3, 2019', '8:30AM', 0, 1, 2, 'example'),
+	(40000, 00004, 'June 4, 2019', '9:30AM', 8, 7, 3, 'example'),
+	(50000, 00005, 'June 5, 2019', '10:30AM', 4, 1, 2, 'example'),
+	(60000, 00006, 'June 6, 2019', '11:30AM', 7, 7, 6, 'example'),
+	(70000, 00007, 'June 7, 2019', '12:30AM', 8, 3, 1, 'example');
+
+insert into tsSurvey values
+        (10000, 00001, 'June 1, 2019', '6:30AM', 'TYPE A'),
```

Comment:
It looks like this file has a bunch of random indents too. I don't know if they are needed but if not I think it would look nicer to have a uniform format

Reply:
I fixed the indents. Kind of forgot about those.